setup.py

```diff
@@ -1,73 +1,102 @@
 #!/usr/bin/env python3
+import io
 import os.path
+import re
 import stat
 import sys
+import tarfile
+import tempfile
+import zipfile
 from distutils.command.build import build as orig_build
 from distutils.core import Command
-from hashlib import sha512
 from http import HTTPStatus
 from urllib.request import urlopen
 
 from setuptools import setup
 from setuptools.command.install import install as orig_install
 
-SHELLCHECK_VERSION = '0.7.0'
+SHELLCHECK_VERSION = '0.7.1'
 PY_VERSION = '1'
 
 
 def get_arch() -> str:
```

Comment:
this function name no longer matches

Reply:
It didn't before this either -- returned `exe` for win32.

Reply:
either way, let's fix it